### ISSUE TITLE
Abnormal Tab with VS Code High Contrast Theme

### ISSUE THREAD

**Issue by Hunter6324** [microsoft/vscode#328914]
Type: <b>Bug</b>

As shown in the attached image, after enabling the **High Contrast Theme** in VS Code, the circled editor tab bar area does not meet the high contrast accessibility requirements:
1.  The background color of the tab title is not pure white.
2.  The tab title text has an unexpected border/box around it, which severely affects readability and visual clarity.

VS Code version: Code 1.131.0 (e4c7e7b1d6d060162f4aa7f8225271b67ce1df75, 2026-07-28T10:51:25Z)
OS version: Windows_NT x64 10.0.26200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i5-1240P (16 x 2112)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x8086, DEVICE=0x46a6 [Intel(R) Iris(R) Xe Graphics], DRIVER_VENDOR=Intel, DRIVER_VERSION=32.0.101.7088 *ACTIVE*<br>GPU1: VENDOR= 0x8086, DEVICE=0x46a6 [Intel(R) Iris(R) Xe Graphics], DRIVER_VERSION=32.0.101.7088<br>GPU2: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.8972<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.64GB (15.74GB free)|
|Process Argv|-n --crash-reporter-id c0a3c5cd-9214-4bd6-9506-761542072d6c|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (27)</summary>

Name|Identifier|Author|Version
---|---|---|---
Markdown Preview Mermaid Support|bierner.markdown-mermaid|bierner|1.32.1
vscode-pandoc|chrischinchilla.vscode-pandoc|ChrisChinchilla|0.7.3
MySQL|cweijan.vscode-mysql-client2|cweijan|9.0.2
GitHub Codespaces|github.codespaces|GitHub|1.18.15
GitHub 存储库|github.remotehub|GitHub|0.64.0
Kilo Code: AI Coding Agent, Copilot, and Autocomplete|kilocode.kilo-code|kilocode|7.4.17
Rainbow CSV|mechatroner.rainbow-csv|mechatroner|3.24.1
Container Tools|ms-azuretools.vscode-containers|ms-azuretools|2.4.5
Chinese (Simplified) (简体中文) Language Pack for Visual Studio Code|ms-ceintl.vscode-language-pack-zh-hans|MS-CEINTL|1.131.2026072717
Python Debugger|ms-python.debugpy|ms-python|2026.6.0
Python|ms-python.python|ms-python|2026.4.0
Pylance|ms-python.vscode-pylance|ms-python|2026.3.1
Python Environments|ms-python.vscode-python-envs|ms-python|1.36.0
开发容器|ms-vscode-remote.remote-containers|ms-vscode-remote|0.466.0
Remote - SSH|ms-vscode-remote.remote-ssh|ms-vscode-remote|0.124.0
Remote - SSH: Editing Configuration Files|ms-vscode-remote.remote-ssh-edit|ms-vscode-remote|0.87.0
WSL|ms-vscode-remote.remote-wsl|ms-vscode-remote|0.104.3
PowerShell|ms-vscode.powershell|ms-vscode|2025.4.0
远程资源管理器|ms-vscode.remote-explorer|ms-vscode|0.5.0
远程存储库|ms-vscode.remote-repositories|ms-vscode|0.42.0
Remote - Tunnels|ms-vscode.remote-server|ms-vscode|1.5.3
VS Code Speech|ms-vscode.vscode-speech|ms-vscode|0.16.0
Chinese (Simplified, China) language support for VS Code Speech|ms-vscode.vscode-speech-language-pack-zh-cn|ms-vscode|0.5.1
open in browser|techer.open-in-browser|techer|2.0.0
vscode-pdf|tomoki1207.pdf|tomoki1207|1.2.2
MDX|unifiedjs.vscode-mdx|unifiedjs|1.8.17
Vue (Official)|vue.volar|Vue|3.3.9

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
binariesv615:30325510
9b8hh234:30694863
nativeloc1:31344060
dwcopilot:31170013
dwoutputs:31242946
copilot_t_ci:31333650
e5gg6876:31282496
pythonrdcb7:31342333
6518g693:31463988
aj953862:31281341
envsactivate1:31551504
cloudbuttont:31379625
3efgi100_wstrepl:31403338
55ia6109:31457054
ec5jj548:31422691
cp_cls_t_966_ss:31526232
4je02754:31466945
c3h7c220:31478652
ge8j1254_inline_auto_hint_haiku:31490510
cp_cls_c_1081:31454833
conptydll_true:31498968
e9c30283:31461165
test_treatment2:31471001
46204921:31447328
control_6dc23131:31555649
idci7584:31464702
e3e4d672:31494082
ei9d7968:31496641
nes-extended-on:31455476
chat:31457767
8hig5102:31480529
89g7j272:31518289
7e187181:31503455
i2gc6536:31499202
52612955:31516516
ghj88844:31499326
ddid_t:31478206
hmra_i5g22:31518061
7df3h592:31512476
cp_cls_t_1082:31535311
logging_enabled_new:31498466
db5d2638:31499441
jb_cp_cls_t_632:31543129
748c7209:31512887
32d76977:31512328
ha629193:31508444
cdk-lw-on:31524445
f9bf7985:31543584
a1ije391_t:31540920
hgf2d445:31529595
8hfba999:31537718
jbcp_cls_pctr_t:31531130
61138546:31518536
cp_intellij_t_nes:31548657
jf4hg949:31526829
nes-curr-leftover-1500:31538366
ahp-both-windows:31556933
h561g132:31533256
chatquotatrajectorynudge-enabled:31548767
hjji5903:31534458
f412h606:31551416
7g2b5551:31542111
5fe6f778:31554331
allow-none:31555437
6i51c363:31551401
ac562147:31560306
c7c27ce7:31554789
nes-gb2-ctrl:31554343
8hc4g144:31558143
1532g621_copy:31554320
treatment-23-1:31555779
unuse_dynamic_mcp:31555281
0d8dfbc3:31560381
switchtoauto:31559321
session-mark-done:31558131

```

</details>

**Comment by Hunter6324** [comment]
<img width="700" height="494" alt="Image" src="https://github.com/user-attachments/assets/773f50de-1820-4d4e-b508-1a8839bb6d51" />

<img width="1400" height="989" alt="Image" src="https://github.com/user-attachments/assets/eb49c65f-0282-4f39-9e06-162d575c03ab" />

**Comment by gjsjohnmurray** [comment]
Try `"workbench.experimental.modernUI": false`

**Comment by Hunter6324** [comment]
> Try `"workbench.experimental.modernUI": false`

ok

**Comment by hawkticehurst** [comment]
@Hunter6324 this was fixed last week via https://github.com/microsoft/vscode/pull/328111. If you update to 1.132.0 you should see the following:

<img width="1014" height="364" alt="Image" src="https://github.com/user-attachments/assets/693483dc-bde4-4e4e-99a2-034113204f18" />
